src/assignments/6-array.js

```diff
@@ -9,6 +9,11 @@ avg=2.5
 
 */
 // start here writing the code for program1
+const array = [1, 4, 2, 3]
+const sum = arr.reduce((partialSum, a) => partialSum + a, 0)
```

Comment:
good that u are using `reduce` method. 
You can use `for loop` to sum the total number in the array also.